CRSP-Tests.postman_collection.json

```diff
@@ -0,0 +1,385 @@
+{
+	"info": {
+		"_postman_id": "6b6d4fdc-9c98-4f74-813f-f029e045bdb8",
+		"name": "CRSP-Tests",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "CRSP Pain001",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"var req = JSON.parse(request.data);",
+							"",
+							"",
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"",
+							"pm.test(\"should have equal incoming and outgoing transaction json\", function () {",
+							"    pm.expect(jsonData.transaction).to.eql(req);",
+							"});",
+							"",
+							"pm.test(\"should send payload to the correct rule processors just once\", function () {",
+							"    var resultsRequiredToBeSend = [",
+							"        \"003@1.0\",",
+							"        \"005@1.0\",",
+							"        \"006@1.0\",",
+							"        \"007@1.0\",",
+							"        \"028@1.0\"",
+							"    ];",
+							"    var rulesSentTo = jsonData.rulesSentTo;",
+							"    const rulesNotSentTo = jsonData.failedToSend",
+							"    const allRules = [...rulesSentTo, ...rulesNotSentTo]",
+							"    for (let i = 0; i < resultsRequiredToBeSend.length; i++) {",
+							"        var checkIfRuleIsSend = (allRules.indexOf(resultsRequiredToBeSend[i]) > -1);",
+							"        console.log(resultsRequiredToBeSend[i] + \" Send was \" + checkIfRuleIsSend);",
+							"        pm.expect(checkIfRuleIsSend).to.eql(true);",
+							"    }",
+							"})",
+							"",
+							"pm.test(\"should send the correct network sub-map\", function () {",
+							"",
+							"    pm.expect(jsonData.networkMap).to.eql",
+							"        (",
+							"           {",
+							"        \"messages\": [",
+							"            {",
+							"                \"id\": \"001@1.0\",",
+							"                \"host\": \"http://openfaas:8080\",",
+							"                \"cfg\": \"1.0\",",
+							"                \"txTp\": \"pain.001.001.11\",",
+							"                \"channels\": [",
+							"                    {",
+							"                        \"id\": \"001@1.0\",",
+							"                        \"host\": \"http://openfaas:8080\",",
+							"                        \"cfg\": \"1.0\",",
+							"                        \"typologies\": [",
+							"                            {",
+							"                                \"id\": \"028@1.0\",",
+							"                                \"host\": \"https://frmfaas.sybrin.com/function/off-typology-processor\",",
```

Comment:
please replace https://frmfaas.sybrin.com with http://gateway.openfaas:8080 (even in the network sub-map) for ALL future tests..........................

Reply:
I've resolved this.